### DISCUSSION TITLE
firefox: cannot use GTK theme

### DISCUSSION THREAD

**Discussion by rieje** [netblue30/firejail#5031] [Q&A] [answered]
I'm trying to get Firefox to use the GTK theme set by my system. The very few issues mentioning GTK themes seems to suggest whitelist: `whitelist ${HOME}/.config/gtk-3.0` in ~/.config/firejail/firefox.local but that has no effect. Version 0.9.68 on Arch Linux. Running without firejail sandbox shows the theme set as intended.

I also included:

    whitelist /usr/share/icons
    whitelist /usr/share/themes

None of themes settings seem particularly sensitive so I would prefer to have Firefox use the GTK theme that's used consistently throughout my system.

**Comment by ghost** [comment, marked as answer]
${HOME}/.config/gtk-3.0 is already whitelisted in whitelist-common.inc, which also whitelists ${HOME}/.themes.
/usr/share/icons and /usr/share/themes are already whitelisted in whitelist-usr-share-common.inc.
Both .inc files get included in firefox-common.profile.
Does your ${HOME}/.config/gtk-3.0/settings.ini actually contain theme item(s) like explained [here](https://coderwall.com/p/no3qfa/setting-gtk2-and-gtk3-theme-via-config-file) for example?

Another option is setting the preferred theme via the GTK_THEME=foo environment variable:

```console
$ grep GTK_THEME ~/.config/firejail/firefox-common.local
env GTK_THEME=Lab16.Lounge

```

**Reply by rieje** [reply]
Thanks, I just didn't have `$GTK_THEME` set.
